tests/unit_tests/converse/test_to_converse_message.py

```diff
@@ -0,0 +1,139 @@
+import pytest
+from aidial_sdk.chat_completion import FunctionCall
+from aidial_sdk.chat_completion import Message as DialMessage
+from aidial_sdk.chat_completion import Role as DialRole
+from aidial_sdk.chat_completion import ToolCall
+
+from aidial_adapter_bedrock.llm.converse.input import to_converse_message
+
+
+@pytest.mark.asyncio
+async def test_to_converse_message_text():
+    dial_message = DialMessage(role=DialRole.USER, content="Hello, world!")
+    converse_message = await to_converse_message(dial_message, storage=None)
+
+    assert converse_message == {
+        "role": "user",
+        "content": [{"text": "Hello, world!"}],
+    }
+
+
+@pytest.mark.asyncio
+async def test_to_converse_message_assistant():
+    dial_message = DialMessage(role=DialRole.ASSISTANT, content="Hello")
+    converse_message = await to_converse_message(dial_message, storage=None)
+
+    assert converse_message == {
+        "role": "assistant",
+        "content": [{"text": "Hello"}],
+    }
+
+
+@pytest.mark.asyncio
+async def test_to_converse_message_function_call_no_content():
+    dial_message = DialMessage(
+        role=DialRole.ASSISTANT,
+        function_call=FunctionCall(
+            name="get_weather", arguments='{"city": "Paris"}'
+        ),
+    )
+    converse_message = await to_converse_message(dial_message, storage=None)
+    assert converse_message == {
+        "role": "assistant",
+        "content": [
+            {
+                "toolUse": {
+                    "toolUseId": "get_weather",
+                    "name": "get_weather",
+                    "input": {"city": "Paris"},
+                }
+            },
+        ],
+    }
+
+
+@pytest.mark.asyncio
+async def test_to_converse_message_function_call_with_content():
+    dial_message = DialMessage(
+        role=DialRole.ASSISTANT,
+        content="Calling a function",
+        function_call=FunctionCall(
+            name="get_weather", arguments='{"city": "Paris"}'
+        ),
+    )
+    converse_message = await to_converse_message(dial_message, storage=None)
+    assert converse_message == {
+        "role": "assistant",
+        "content": [
+            {"text": "Calling a function"},
+            {
+                "toolUse": {
+                    "toolUseId": "get_weather",
```

Comment:
Please add a test with two function calls calling the same function with different arguments: e.g. get_weather(city=Paris) and get_weather(city=London). It will test the generation of **unique** `toolUseId` identifiers.